### DISCUSSION TITLE
How can we specify a Name for a manually created holding?

### DISCUSSION THREAD

**Discussion by karter16** [ghostfolio/ghostfolio#2523] [Q&A] [answered]
How can we specify a name for a manually created holding?

We can see in the below screenshot that RKLB (Rocket Lab USA Inc.) has a name and shows the code below it.

Whereas when creating manual holdings it seems we can only specify the "code" and not the "name"?

<img width="1669" alt="Screenshot 2023-10-21 at 5 39 27 PM" src="https://github.com/ghostfolio/ghostfolio/assets/4284373/abb5a507-f52a-47a2-bd8f-d790c6419c15">

This causes annoyances when it comes to the holding screen:

<img width="1444" alt="Screenshot 2023-10-21 at 5 41 54 PM" src="https://github.com/ghostfolio/ghostfolio/assets/4284373/d1868656-9583-4b13-9e8d-1bab8a9a5405">

Is there any way to do this via the UI, or am I going to need to dive into the database?

**Comment by dtslvr** [comment, marked as answer]
Hi @karter16

The feature to create manual asset profiles is quite new. At the moment you need to adjust the name directly in the database (_SymbolProfile_ table). However, I have created a ticket (#2524) to make it possible directly in the app.

**Reply by karter16** [reply]
awesome thanks for the confirmation :-)
